src/mono/mono.proj

```diff
@@ -32,9 +32,29 @@
         <_MonoExtraCFLAGS>-O2 -g</_MonoExtraCFLAGS>
         <_MonoExtraCXXFLAGS>-O2 -g</_MonoExtraCXXFLAGS>
     </PropertyGroup>
+    <PropertyGroup Condition=" '$(TargetArchitecture)' == 'arm64' ">
+        <_MonoTuple>aarch64-linux-gnu</_MonoTuple>
+    </PropertyGroup>
+    <PropertyGroup Condition=" '$(TargetArchitecture)' == 'arm' ">
+        <_MonoTuple>arm-linux-gnueabihf</_MonoTuple>
+    </PropertyGroup>
+    <PropertyGroup Condition=" '$(TargetArchitecture)' == 'arm' Or '$(TargetArchitecture)' == 'arm64' ">
+        <_MonoCrossDir>$(ROOTFS_DIR)</_MonoCrossDir>
+        <_MonoExtraCFLAGS>$(_MonoExtraCFLAGS) --sysroot=$(_MonoCrossDir) --target=$(_MonoTuple)</_MonoExtraCFLAGS>
```

Comment:
what is the purpose of `--target=$(_MonoTuple)` in `CFLAGS` and friends?

Reply:
Clang uses a `--target` flag for cross-compiling (rather than GCC where you have a `tuple-gcc` binary to emit for that target)

Reply:
Would be nice to somehow use either.